modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -107,8 +125,10 @@ export default class ColorInput extends React.Component<ColorInputProps> {
     const formattedValue = this.formatValue(value);
 
     return (
-      <ColorInputContainer>
+      <ColorInputContainer 
+      {...elemProps}>
```

Comment:
Why were `elemProps` added here? This is a potentially breaking change

Reply:
Looks like it is not useful, I might have done that to debug something.